### DISCUSSION TITLE
Import from Postman

### DISCUSSION THREAD

**Discussion by I525970** [hoppscotch/hoppscotch#4163] [Q&A] [answered]
Hi everyone~ I try to migrate scripts form Postman, but there is no test content when Imported with Postman scripts. Pls let me know how to import from Postman script with completed content. Thanks!

**Comment by jamesgeorge007** [comment, marked as answer]
Hi, we’ve added support for importing scripts from Postman collections (see #5417) in the [latest release](https://github.com/hoppscotch/hoppscotch/releases/tag/2025.10.0), as part of ongoing scripting improvements based on RFC #5221. Please note that this feature is experimental and requires opting into the `Experimental scripting sandbox` option via settings.

<img width="1485" height="529" alt="Image" src="https://github.com/user-attachments/assets/a7e9ec55-af17-4763-8062-c9bee9891b71" />

Closing this issue. We welcome your feedback, which will help shape the next generation of Hoppscotch scripting. Feel free to share it via the RFC.
